<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdltmSqlView</class>
 <widget class="QWidget" name="SdltmSqlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ExtendedTableWidget" name="table" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="navigateCtrl" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QToolButton" name="buttonBegin">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scroll to the beginning&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clicking this button navigates to the beginning in the table view above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>|&lt;</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons/icons.qrc">
             <normaloff>:/icons/resultset_first</normaloff>:/icons/resultset_first</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonPrevious">
           <property name="toolTip">
            <string>Scroll one page upwards</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clicking this button navigates one page of records upwards in the table view above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons/icons.qrc">
             <normaloff>:/icons/resultset_previous</normaloff>:/icons/resultset_previous</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelRecordset">
           <property name="text">
            <string>0 - 0 of 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonNext">
           <property name="toolTip">
            <string>Scroll one page downwards</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clicking this button navigates one page of records downwards in the table view above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons/icons.qrc">
             <normaloff>:/icons/run</normaloff>:/icons/run</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonEnd">
           <property name="toolTip">
            <string>Scroll to the end</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clicking this button navigates up to the end in the table view above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>&gt;|</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons/icons.qrc">
             <normaloff>:/icons/run_line</normaloff>:/icons/run_line</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonGoto">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to jump to the specified record&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This button is used to navigate to the record number specified in the Go to area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Go to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editGoto">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Enter record number to browse</string>
           </property>
           <property name="whatsThis">
            <string>Type a record number in this area and click the Go to: button to display the record in the database view</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedTableWidget</class>
   <extends>QWidget</extends>
   <header>extendedtablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
